src/explore-education-statistics-common/src/components/form/FormWysiwygArea.tsx

```diff
@@ -0,0 +1,59 @@
+import React, {
+  ChangeEventHandler,
+  KeyboardEventHandler,
+  MouseEventHandler,
+  ReactNode,
+} from 'react';
+import ErrorMessage from '../ErrorMessage';
+import EditableMarkdownRenderer from '../../../../explore-education-statistics-admin/src/modules/find-statistics/components/EditableMarkdownRenderer';
+
+export interface FormWysiwygAreaProps {
+  error?: ReactNode | string;
+  hint?: string;
+  id: string;
+  toolbarStyle?: 'full' | 'reduced';
+  label: ReactNode | string;
+  source: string;
+  name: string;
+  onChange?: ChangeEventHandler<HTMLTextAreaElement>;
+  onKeyPress?: KeyboardEventHandler<HTMLTextAreaElement>;
+  onClick?: MouseEventHandler<HTMLTextAreaElement>;
+  onContentChange: (content: string) => void;
+  rows?: number;
+  value?: string;
+  defaultValue?: string;
+  list?: string;
+  additionalClass?: string;
+}
+
```

Comment:
If we're spreading `{...props}` onto the EditableMarkdownRenderer, should the `FormWysiwygAreaProps` extend `EditableMarkdownRendererProps` with { label, hint, etc...  } ?